src/js/page_sections/Contact.js

```diff
@@ -6,61 +6,76 @@ import Hours from 'js/modules/Hours';
 
 class Contact extends Component {
 
+  cleanContact(contact) {
+    let cleaned = Object.assign(contact.contact);
+    if(cleaned.hours) {
+      cleaned.hours = cleaned.hours.edges.map(({ node: hours }) => hours);
+    }
+    return cleaned;
+  }
+
   render() {
 
     const { contacts } = this.props;
-
-    return (contacts) && (
-
-      <div className="coa-section">
-        <SectionTitle title="Contact" noBorder={true} />
-
-        {
-          contacts.map((contact, index) => {
-
-            const name = get(contact, "name", null);
-            const phone = get(contact, "phone", null);
-            const email = get(contact, "email", null);
-            const location = get(contact, "location", null);
-            const hours = get(contact, "hours", null);
-
-            return (
-              <div key={index} className="coa-section__subsection">
-
-                  <SectionSubtitle title={name} />
-
-                { phone && (
-                  <div className="coa-section__map">
-                    <h5>Phone Number</h5>
-                    <a href={`tel:${phone}`}>{phone}</a>
-                  </div>
-                )}
-
-                { email && (
-                  <div className="coa-section__map">
-                    <h5>Email</h5>
-                    <a href={`mailto:${email}`}>{email}</a>
-                  </div>
-                )}
-
-                { location && (
-                  <div className="coa-section__map">
-                    <h5>Address</h5>
-                    <span>{location.street}</span>
-                    <span>{location.city}, {location.state} {location.zip}</span>
-                    <span>{location.country}</span>
-                  </div>
-                )}
-
-                <Hours hours={hours} />
-
-              </div>
-            );
-
-          })
-        }
-      </div>
-    );
+    let JSX;
+
+    if (!contacts || !contacts.length) {
+      JSX = null;
+    } else {
+      JSX = (
```

Comment:
this is a nifty pattern I hadn't really seen before

Reply:
it's my way of getting around linter requirements that always expect return statements.